recipes/clhep/all/conanfile.py

```diff
@@ -8,14 +8,14 @@
 
 required_conan_version = ">=1.53.0"
 
-
 class ClhepConan(ConanFile):
     name = "clhep"
     description = "Class Library for High Energy Physics."
     license = "LGPL-3.0-only"
     topics = ("cern", "hep", "high energy", "physics", "geometry", "algebra")
     homepage = "http://proj-clhep.web.cern.ch/proj-clhep"
     url = "https://github.com/conan-io/conan-center-index"
+    implements = ["auto_shared_fpic"]
```

Comment:
I love that you want to add such new features to CCI, but sadly it will have to wait until we stop supporting v1 alongside v2, you still need to manually implement this by hand :)